packages/wp-dashboard/src/components/editorSettings/googleAnalytics/index.js

```diff
@@ -247,97 +275,110 @@ function GoogleAnalyticsSettings({
         )}
       </div>
       <div>
-        {analyticsActive ? (
-          <TextInputHelperText size={TextSize.Small}>
-            {TEXT.SITE_KIT_IN_USE}
-          </TextInputHelperText>
-        ) : (
+        <InlineForm>
+          <VisuallyHiddenLabel htmlFor="gaTrackingId">
+            {TEXT.ARIA_LABEL}
+          </VisuallyHiddenLabel>
+          <SettingsTextInput
+            aria-label={TEXT.ARIA_LABEL}
+            id="gaTrackingId"
+            value={analyticsId}
+            onChange={onUpdateAnalyticsId}
+            onKeyDown={handleOnKeyDown}
+            placeholder={TEXT.PLACEHOLDER}
+            hasError={Boolean(inputError)}
+            hint={inputError}
+          />
+          <SaveButton
+            type={ButtonType.Secondary}
+            size={ButtonSize.Small}
+            disabled={disableSaveButton}
+            onClick={handleOnSave}
+          >
+            {TEXT.SUBMIT_BUTTON}
+          </SaveButton>
+        </InlineForm>
+        <TextInputHelperText size={TextSize.Small}>
+          <TranslateWithMarkup
+            mapping={{
+              a: (
+                <InlineLink
+                  href={TEXT.CONTEXT_LINK}
+                  rel="noreferrer"
+                  target="_blank"
+                  size={TextSize.Small}
+                  onClick={onContextClick}
+                />
+              ),
+            }}
+          >
+            {TEXT.CONTEXT}
+          </TranslateWithMarkup>
+        </TextInputHelperText>
+        {analyticsActive && (
           <>
-            <InlineForm>
-              <VisuallyHiddenLabel htmlFor="gaTrackingId">
-                {TEXT.ARIA_LABEL}
-              </VisuallyHiddenLabel>
-              <SettingsTextInput
-                aria-label={TEXT.ARIA_LABEL}
-                id="gaTrackingId"
-                value={analyticsId}
-                onChange={onUpdateAnalyticsId}
-                onKeyDown={handleOnKeyDown}
-                placeholder={TEXT.PLACEHOLDER}
-                hasError={Boolean(inputError)}
-                hint={inputError}
-                disabled={analyticsActive}
-              />
-              <SaveButton
-                type={ButtonType.Secondary}
-                size={ButtonSize.Small}
-                disabled={disableSaveButton}
-                onClick={handleOnSave}
-              >
-                {TEXT.SUBMIT_BUTTON}
-              </SaveButton>
-            </InlineForm>
             <TextInputHelperText size={TextSize.Small}>
+              <TranslateWithMarkup>
+                {__('<b>Note: </b>', 'web-stories')}
+              </TranslateWithMarkup>
+              {TEXT.SITE_KIT_IN_USE}
+            </TextInputHelperText>
```

Comment:
The `Note:` part should be either part of `TEXT.SITE_KIT_IN_USE` or we should use `sprintf`. No string concatenation.